observability/akka-http-metrics/src/main/scala/com/daml/metrics/akkahttp/LabelExtractor.scala

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.metrics.akkahttp
+
+import akka.http.scaladsl.model.{HttpRequest, HttpResponse}
+
+object LabelExtractor {
+
+  def labelsFromRequest(request: HttpRequest): Seq[(String, String)] = {
+    val baseLabels = Seq[(String, String)](
+      ("http_verb", request.method.name),
```

Comment:
I'd strongly suggest defining these magic constants for the labels we generate in a single place and referencing them from there.